Makefile

```diff
@@ -48,26 +48,6 @@ run-local-cloud:
 	./meshery; \
 	cd ..
 
-run-local-cloud-noauth:
-	cd cmd; go clean; rm meshery; go mod tidy; go build -tags draft -a -o meshery; \
-	SAAS_BASE_URL="http://mesherylocal.layer5.io:9876" \
-	PORT=9081 \
-	DEBUG=true \
-	NO_AUTH=true \
-	ADAPTER_URLS=$(ADAPTER_URLS) \
-	./meshery; \
-	cd ..
-
```

Comment:
@girishranganathan do these go away because, there is no local cloud anymore? Only local provider?

Reply:
Local cloud still exists. It is if you want to run Meshery-cloud on your local and want Meshery to point to it.